.github/workflows/builds-and-tests.yml

```diff
@@ -66,7 +67,20 @@ jobs:
             codecov: NO
             mpi: parallel
             build-system: cmake
-    name: ${{ matrix.os }}-${{ matrix.target }}-${{ matrix.mpi }}-${{ matrix.build-system }}
+            hypre-target: basic
+          - os: ubuntu-18.04
+            target: optim
+            codecov: YES
+            mpi: parallel
+            build-system: make
+            hypre-target: basic
```

Comment:
The is the job with a basic hypre build. Although I realize it may be redundant with the one already present in the matrix.